test/correctness/simd_op_check.cpp

```diff
@@ -192,17 +193,17 @@ struct Test {
         // Define a vectorized Func that uses the pattern.
         Func f(name);
         f(x, y) = e;
-        f.bound(x, 0, W).vectorize(x, vector_width);
+        f.bound(x, PAD, W-PAD).vectorize(x, vector_width);
```

Comment:
I still don't think any of these changes should be here (no changes to the schedule/RDom parameters). I think the only change should be on https://github.com/halide/Halide/pull/2050/files#diff-f255460cd912e3d8bad7647aeaab78daL109, change the min from 0 to -PAD. It looks like apps/simd_op_check already has PAD worth of padding on each side of the image (1024 vs 768).